### ISSUE TITLE
Better Combat Offhand Attacking

### ISSUE THREAD

**Issue by SomeRandomPetey** [iwoplaza/MoBends#59]
When using the Better Combat Rebirth mod, if you try to attack in your offhand it uses the main hand attack animation. It would be nice if there was a left hand swing animation as well.

**Comment by QuerrixTU** [comment]
yes it would be cool and i think it's easy to add iwoplaza already do the animation for the right hands

**Comment by Roboslime** [comment]
Yeah, all that would be added to the editor is a state for offhand attack...I really want this
